templates/documents/collection_file_status.html

```diff
@@ -0,0 +1,21 @@
+<div id="file-status-{{ collection_file.id }}"
+     class="tooltip"
+     data-tip="{{ collection_file.status_enum.label }}"
+     {% if collection_file.status == 'pending' or collection_file.status == 'in_progress' %}
+       hx-get="{% url 'documents:get_collection_file_status' team.slug collection.id collection_file.id %}"
+       hx-trigger="every 5s"
+       hx-swap="outerHTML"
+     {% endif %}>
+  {% if collection_file.status == 'pending' %}
+    <span class="status"></span>
+  {% elif collection_file.status == 'in_progress' %}
+    <div class="inline-grid *:[grid-area:1/1]">
+      <div aria-label="warning" class="status status-warning animate-ping"></div>
```

Comment:
nit: I think the ping looks great, but we do use the spinners elsewhere so maybe opt for a spinner?

Also aria label of "in progress" would be more accurate

Reply:
We can do that yes. here's a mini demo of what that looks like

![demo](https://github.com/user-attachments/assets/3a6985d4-9e66-473a-9b5a-38b17e0d1040)